domain/src/main/java/nl/tehdreamteam/se42/domain/message/ImageMessage.java

```diff
@@ -0,0 +1,40 @@
+package nl.tehdreamteam.se42.domain.message;
+
+import nl.tehdreamteam.se42.domain.Message;
+import nl.tehdreamteam.se42.domain.User;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import java.util.Arrays;
+
+/**
+ * @author Oscar de Leeuw
+ */
+@Entity
+public class ImageMessage extends Message<Byte[]> {
+
+    @Column(name = "image")
+    private Byte[] imageContent;
+
+    protected ImageMessage() {
+
+    }
+
+    public ImageMessage(User sender, Byte[] content) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)